compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -181,12 +186,12 @@ CircleConst *create_circleconst(GraphBuilderContext *context, int32_t tensor_ind
         break;
 
       case loco::DataType::STRING:
-        copy_data<loco::DataType::STRING>(buffer, num_elements, const_node);
+        copy_string(buffer, num_elements, const_node);
```

Comment:
I don't think this is related with `Use direct access API`, or is it?

Reply:
Yes, let me reduce this change too.

Reply:
Done.